### Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек.

Какова вероятность, что ни одна из них не перегорит в первый день? Какова вероятность, что перегорят ровно две?

Для решения используем частный случай биномального распределения - формулу Пуассона, так как:

- вероятность отдельного испытания слишком мала, менее 0.01%
- Опытов слишком много, более 1000

Формула Пуассона - P(m) = (lambda^m / m!) * e^(-lambda)

Необходимо определить среднее количество поступления события (lambda = p * n)

В данной задаче:

p = 0.0004

n = 5000

In [1]:
import math 

lambdaa = 0.0004 * 5000
lambdaa

2.0

2. Необходимо подставить значение m, как 0 и как 2 соответственно.

P(0) = (2^0 / 0!) * e^(-2)

P(2) = (2^2 / 2!) * e^(-2)

In [2]:
def p(lambdaa, m=0):
    return ((lambdaa ** m) / math.factorial(m)) * (math.e ** -lambdaa)


p_0 = p(lambdaa)
p_2 = p(lambdaa, m=2)

p_0, p_2

(0.1353352832366127, 0.2706705664732254)

Ответ: Вероятность, что ни одна из лампочек не перегорит в первый же день = 13.5%, а того, что перегорит ровно 2 = 27%